content/developer/reference/backend/views.rst

```diff
@@ -1164,6 +1164,9 @@ take the following attributes:
   if set to true, the gantt view will start at the first record,
   instead of starting at the beginning of the year/month/day.
 
+``pill_label``
+  if set to true, the time will appear in the pill label when the scale is week or month. (eg: ``'7:00 AM - 11:00 AM (4h) - DST Task 1'``)
```

Comment:
- I think it's "_on_ the pill label"
- "is set on week"
- e.g.,

None of these following points are respected in the rest of this document, so it's not blocking... :)

- Start with a capital "I".
- it's best to write in the present tense and not in the future tense (cfr [guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#grammatical-tenses) )
- new line at the 100th character (cfr [RST guidelines](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character) )
- writing in literal only requires one ` (see [RST cheat sheet](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#technical-term-literal)). I don't comment it for line 1172 because it would break the consistency of the page's code.

Question: why do you write `'` before and after `7:00 AM ..... Task 1`?

```suggestion
  If set to true, the time appears in the pill label when the scale is set on week or month. (e.g.,
  `7:00 AM - 11:00 AM (4h) - DST Task 1`)
```

Reply:
> * I think it's "_on_ the pill label"
> * "is set on week"
> * e.g.,
> 
> None of these following points are respected in the rest of this document, so it's not blocking... :)
> 
> * Start with a capital "I".
> * it's best to write in the present tense and not in the future tense (cfr [guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#grammatical-tenses) )
> * new line at the 100th character (cfr [RST guidelines](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character) )
> * writing in literal only requires one ` (see [RST cheat sheet](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#technical-term-literal)). I don't comment it for line 1172 because it would break the consistency of the page's code.
> 
> Question: why do you write `'` before and after `7:00 AM ..... Task 1`?


Ans-No special purpose. Now I have removed it.